live-examples/js-examples/promise/promise-any.js

```diff
@@ -0,0 +1,9 @@
+const promise1 = Promise.reject(0);
+const promise2 = new Promise((resolve) => setTimeout(resolve, 100, 'quick'));
+const promise3 = new Promise((resolve) => setTimeout(resolve, 500, 'slow'));
+const promises = [promise1, promise2, promise3];
```

Comment:
Super-nit: I'd like a blank line between lines 3 and 4, to separate the three promise declarations from the array declaration.